app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -951,6 +955,37 @@ class DiscoveryView extends React.Component {
     this.setState({ mapLocationData: clusteredData, mapLevel });
   };
 
+  // Cloned from Samples.jsx (deprecated)
```

Comment:
Why is this deprecated?

Reply:
@tfrcarvalho deleted this comment b/c it was confusing and added a comment to the setItem below 👍